ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -190,9 +190,9 @@
             <date-part name="year" suffix=", "/>
           </date>
           <group>
-            <text term="volume" form="short" suffix=". "/>
-            <text variable="number-of-volumes" prefix=". " suffix="/"/>
-            <text variable="volume" suffix=","/>
+              <text term="volume" form="short" suffix=".&#160;"/>
+            <text variable="volume"  suffix="/"/>
```

Comment:
this seems like you'd want to do it with a delimiter, i.e.

```
<group delimiter="/" suffix=", "/>
  <text variable="volume"/>
  <text variable="number-of-volumes/>
</group>
```

Also, same issue with extra space.